README.md

```diff
@@ -29,7 +47,7 @@ $ oc apply -f deploy/${DISTRO}/image.yaml
 ```
 Note: For UBI build to work you need to add a subscription entitlement key
 ```
-oc create secret generic etc-pki-entitlement --from-file=entitlement.pem=path/to/pem/file/{id}.pem --from-file=entitlement-key.pem=path/to/pem/file/{id}.pem
+$ oc create secret generic etc-pki-entitlement --from-file=entitlement.pem=path/to/pem/file/{id}.pem --from-file=entitlement-key.pem=path/to/pem/file/{id}.pem
 
 ```
 https://docs.openshift.com/container-platform/4.3/builds/running-entitled-builds.html#builds-source-secrets-entitlements_running-entitled-builds
```

Comment:
Add some context around this link for documentation purposes

Reply:
Done